.golangci.yml

```diff
@@ -1,110 +1,105 @@
+version: "2"
 run:
   go: "1.23"
-  timeout: 1m
   tests: true
-
-issues:
-  include:
-    - "EXC0014" # Enable revive export rule with "should start with"
-  exclude-rules:
-    # Exclude other linters by default on tests files
-    # we only run testifylint https://github.com/Antonboom/testifylint on test files
-    - path: _test\.go$
-      linters:
-        - errcheck
-        - gci
-        - gofumpt
-        - gosec
-        - gosimple
-        - govet
-        - ineffassign
-        - misspell
-        - staticcheck
-        - unused
-        - usestdlibvars
-        - whitespace
-        - revive
-        - gocritic
-
 linters:
-  disable-all: false
   enable:
-    - errcheck
-    - gci
     - gocritic
-    - gofumpt
     - gosec
-    - gosimple
-    - govet
-    - ineffassign
     - misspell
     - revive
-    - staticcheck
     - testifylint
     - unconvert
-    - unused
     - usestdlibvars
     - whitespace
-
-linters-settings:
-  revive:
+  settings:
+    gocritic:
+      enable-all: true
+      disabled-checks:
+        - appendAssign
+        - hugeParam
+        - rangeValCopy
+        - regexpSimplify
+    govet:
+      disable:
+        - shadow
+        - fieldalignment
+      enable-all: true
+    misspell:
+      locale: US
+    revive:
+      rules:
+        - name: blank-imports
+        - name: context-as-argument
+          arguments:
+            - allowTypesBefore: '*testing.T'
+        - name: context-keys-type
+        - name: dot-imports
+        - name: empty-block
+        - name: error-naming
+        - name: error-return
+        # - name: error-strings
+        - name: errorf
+        # - name: exported
+        #   arguments:
+        #     - checkPrivateReceivers
+        #     - sayRepetitiveInsteadOfStutters
```

Comment:
Why disabling exported and error-strings ?

Do you plan to reactivate them later?

Reply:
For exported, it's because the `comments` in `presets` is missing.

See my suggestion [here](https://github.com/go-fuego/fuego/pull/464#discussion_r2036796243)

I'm pretty sure the EXC0014 exclusion I added months ago led `golangci-lint migrate` to remove all comments exclusions that were present by default in golangci-lint v1

Please add it to presets, so you can enable exported again.

For errors-return I think it's simply because the linter evolved